src/components/map/MapService.js

```diff
@@ -968,87 +974,14 @@ goog.require('ga_urlutils_service');
-                subLayersIds: [
-                  'ch.swisstopo.swisstlm3d-karte-farbe.3d'
-                ],
-                attribution: 'tlm farbe 3D',
-                attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
-                    'swisstopo/en/home/products/height/swissALTI3D.html'
-              };
-              // WMTS (not MapProxy)
-              response.data['ch.swisstopo.swissimage-product_3d'] = {
-                serverLayerName: 'ch.swisstopo.swissimage-product',
-                timestamps: ['20151231_50'],
-                url: '//wmts{s}.geo.admin.ch/1.0.0/{Layer}/default/' +
-                    '{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}',
-                subdomains: '56789',
-                attribution: 'swissimage 3D',
-                attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
-                    'swisstopo/en/home/products/height/swissALTI3D.html',
-                has3dMetadata: true
-              };
-              response.data['ch.swisstopo.swisstlm3d-karte-farbe.3d'] = {
-                type: 'wmts',
-                format: 'jpeg',
-                serverLayerName: 'ch.swisstopo.swisstlm3d-karte-farbe.3d',
-                timestamps: ['20150501_50'],
-                url: '//wmts{s}.geo.admin.ch/1.0.0/{Layer}/default/' +
-                    '{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}',
-                subdomains: '56789',
-                attribution: 'swisstlm 3D Farbe',
-                attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
-                    'swisstopo/en/home/products/height/swissALTI3D.html',
-                extent: [481500, 72250, 835750, 297800],
-                has3dMetadata: true
-              };
-              response.data['ch.swisstopo.swisstlm3d-karte-grau.3d'] = {
-                type: 'wmts',
-                format: 'jpeg',
-                serverLayerName: 'ch.swisstopo.swisstlm3d-karte-grau.3d',
-                timestamps: ['20150501_50'],
-                url: '//wmts{s}.geo.admin.ch/1.0.0/{Layer}/default/' +
-                    '{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}',
-                subdomains: '56789',
-                attribution: 'swisstlm 3D Grau',
-                attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
-                    'swisstopo/en/home/products/height/swissALTI3D.html',
-                has3dMetadata: true
-              };
-
               // Terain
               response.data['ch.swisstopo.terrain.3d'] = {
                 type: 'terrain',
                 serverLayerName: 'ch.swisstopo.terrain.3d',
                 timestamps: ['20151231'],
-                attribution: 'swisstopo 3D',
+                attribution: 'swisstopo',
                 attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
-                    'swisstopo/en/home/products/height/swissALTI3D.html'
+                    'swisstopo/en/home.html'
```

Comment:
Ok, `$translate.uses()` should be enough as a new layersConfig is performed on language change.